test/CMakeLists.txt

```diff
@@ -57,6 +57,13 @@ add_custom_target(run-onedpl-tests
     DEPENDS build-onedpl-tests
     COMMENT "Build and run all oneDPL tests")
 
+# Disable iterators check on Windows
+# For details please see https://learn.microsoft.com/en-us/cpp/standard-library/iterator-debug-level?view=msvc-170
+if (WIN32)
+    add_definitions(-D_ITERATOR_DEBUG_LEVEL=0)
+    message(STATUS "Disable iterators check on Windows : #define _ITERATOR_DEBUG_LEVEL 0")
```

Comment:
It seems like we may want to change this to: 
`# Disable checked iterators on Windows`
and 
`message(STATUS "Disable checked iterators on Windows : #define _ITERATOR_DEBUG_LEVEL 0")`

As Microsoft describes this setting as enabling or disabling "checked iterators" rather than "iterators check" (the setting we are using to avoid the variadic function call).

Reply:
Fixed